src/nvim/eval/funcs.c

```diff
@@ -9135,7 +9118,8 @@ static void f_setreg(typval_T *argvars, typval_T *rettv, FunPtr fptr)
 
   if (set_unnamed) {
     // Discard the result. We already handle the error case.
-    if (op_reg_set_previous(regname)) { }
+    if (op_reg_set_previous(regname)) {
+    }
```

Comment:
off-topic: why isn't this...
```suggestion
    (void)op_reg...
```